packages/desktop/components/popups/AllotManaPopup.svelte

```diff
@@ -34,7 +36,7 @@
         $selectedWallet?.hasConsolidatingOutputsTransactionInProgress || $selectedWallet?.isTransferring
     $: asset = $visibleSelectedWalletAssets[$activeProfile?.network?.id].mana
     $: validAmount = Big(rawAmount ?? 0)?.gt(0)
-    $: accountAddress, validAmount, void preparedOutput()
+    $: accountAddress, validAmount, amount, void preparedAllotMana()
```

Comment:
if you make this based on amount too, you should debounce it otherwise we owuld be bombarding `preparedAllotMana` when using the slider. I suggest to add a debouncer like we have in the delegation popup